specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -102,7 +110,6 @@ swagger-to-sdk:
       - node sdkauto_afterscript.js kubernetesconfiguration/resource-manager
 ```
 
-
```

Comment:
All SDK generation readmes need to be updated for the new API version. Default tag is only used for `python` and `typescript`. Rest need to have additions so that the new api package tag is included